app/models/container_label_tag_mapping.rb

```diff
@@ -84,12 +84,12 @@ def self.create_tag(name, value, category_tag)
                                                  :tag         => category_tag)
     end
 
-    if value.empty?
-      # ':' character passes validation but won't be produced by sanitize_name.
-      entry = category.add_entry(:name => ':empty:', :description => '<empty value>')
-    else
-      entry = category.add_entry(:name => Classification.sanitize_name(value), :description => value)
-    end
+    entry = if value.empty?
+              # ':' character passes validation but won't be produced by sanitize_name.
+              category.add_entry(:name => ':empty:', :description => '<empty value>')
+            else
+              category.add_entry(:name => Classification.sanitize_name(value), :description => value)
+            end
```

Comment:
Pushed a change.
I actually liked best the original 2 calls to add_entry — the argument names compactly explained what's what — but Rubocop objected.